components/centraldashboard/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubeflow Landing-Page
+This component is the dashboard for Kubeflow deployents
+
+## Getting Started
+For devmode, do these steps to spin up a stand-alone server locally:
+- Clone the repo
```

Comment:
I'd just say you need npm, then you can do npm install in this directory, bower install, then serve the static files using any server.
Even better, how about including `bower install` in npm's post-install script, and including "serve" as a dev dependency, such that doing `npm start` serves the files using it?

Reply:
This will happen with the new issue: #2512. For now, this is the best set of instructions for the site to load in dev mode

Reply:
This is more about simplifying the UI requirements than the backend. Devs should not be required to install bower globally (and managing its version) if it can be done as part of npm. And owning an easy way to serve the static file is always a good idea.

This is a nit though, I won't block on it.